core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -18,8 +20,16 @@ object ErrorValidator {
     }
   }
 
+  private def ideMessageMatchesMessagesFromConfig(ideMessage: IdeMessage, messagesFromConfig: Seq[String]): Boolean =
+    messagesFromConfig.exists { configMessage =>
+      trimEachLine(ideMessage.content).contains(trimEachLine(configMessage)) ||
+      configMessage.r.findFirstIn(ideMessage.content).nonEmpty
+    }
+
+  private def trimEachLine(s: String): String = s.split('\n').map(_.trim).mkString("\n")
```

Comment:
there is `linesIterator` method

Reply:
sure, and what would be the benefit from using it instead of current logic?

Reply:
well, it was created for the purpose of splitting string into lines. It is lazy and likely covers windows/linux separators. Also it names clearly shows what you mean. Even if you don't need these properties, why reimplement it if it is there?

Reply:
ok, done as suggested